python/ray/tune/tests/test_tuner_restore.py

```
@@ -397,17 +391,14 @@ def test_tuner_resume_unfinished(ray_start_2_cpus, tmpdir, monkeypatch):
 
     results = tuner.fit()
     assert len(results) == 4
-    assert len(results.errors) == 1
+    # assert len(results.errors) == 1
```

why we ignored the error here?

---

Fixed.